guides/doc-Provisioning_Guide/topics/configuring_the_discovery_service.adoc

```diff
@@ -0,0 +1,39 @@
+[[configuring-the-discovery-service]]
+== Configuring the Discovery Service
+
+{ProjectName} provides a method to automatically detect hosts on a network that are not in your {Project} inventory. These hosts boot the discovery image that performs hardware detection and relays this information back to {ProjectServer}. This method creates a list of ready-to-provision hosts in {ProjectServer} without needing to enter the MAC address of each host.
+
+When you configure the Discovery service, and boot a blank, bare-metal host, the boot menu has two options: `local` and `discovery`. If you select `discovery` to boot the Discovery image, after a few minutes, the Discovery image completes booting and a status screen is displayed.
+
+The Discovery service is enabled by default on {ProjectServer}. However, the default setting of the global templates is to boot from the local hard drive. To use discovery you must change the default entry in the template to discovery.
+
+image::PXE-mode.png[]
+
+To use {ProjectServer} to provide the Discovery image, install the following RPM packages:
+
+ifeval::["{build}" == "foreman"]
+* `tfm-rubygem-foreman_discovery`
+endif::[]
+* `foreman-discovery-image`
+* `rubygem-smart_proxy_discovery`
```

Comment:
This should be:

```
* `tfm-rubygem-foreman_discovery`
ifeval::["{build}" == "satellite"]
* `foreman-discovery-image`
endif::[]
* `rubygem-smart_proxy_discovery`
```

Both plugins ship with upstream and downstream, however FDI is not distributed as RPM in upstream due to technical constraints we have on our build server.